### ISSUE TITLE
how can I reach the same accuracy in the paper?

### ISSUE THREAD

**Issue by coallar** [chenjun2hao/DDRNet.pytorch#26]
Hi, bro, I train DDRNet-23 without pretrained model using 1024x1024 resolution. While it get 0.71mIoU metric.
I want to know how can I reach the same accuracy in the paper? what is your trick? thanks
